front/src/common/authorization/components/PermissionBasedContent.tsx

```diff
@@ -0,0 +1,28 @@
+import useUserPermissionsCheck from '../hooks/useUserPermissionsCheck';
+
+export type PermissionBasedContentProps = {
```

Comment:
No need to export the type
```suggestion
type PermissionBasedContentProps = {
```